theme/layouts/_header.ejs

```diff
@@ -0,0 +1,27 @@
+<header>
+  <div class="tablet-hide">
+    <div class="col2">
+      <a href="\">
```

Comment:
Meeting notes: Prevent anchor taking whole width.

Reply:
What about a `display: block;` wrapper?

Reply:
I think it's already done by adding `.header-top`, you just need to remove `width` prperty.